cql2pgjson/src/main/java/org/folio/cql2pgjson/util/Cql2SqlUtil.java

```diff
@@ -37,10 +37,7 @@ private Cql2SqlUtil() {
   @SuppressWarnings("squid:S3776")  // suppress "Cognitive Complexity of methods should not be too high"
   public static String cql2like(String s) {
     StringBuilder like = new StringBuilder();
-    /**
-     * true if the previous character is an escaping backslash
-     */
-    boolean backslash = false;
+    boolean backslash = false;  // previous character is backslash
```

Comment:
Can you re-insert "escaping" before backslash?
Can you re-insert "true if" at the begin of the comment to show that the comment is not about "false".